### DISCUSSION TITLE
AI只是曇花一現？

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2070] [General]
https://www.cw.com.tw/article/5131609
Ai一直是最近熱門的議題，然而openAI今年的虧損卻高達40億美元，這讓我不禁思考AI真的是未來的方向嗎？
OpenAI今年獲利約45億美元，然而由於市場對 AI 需求暴增，企業對於技術的要求不斷提升，讓 OpenAI 需要大量投入資金研發，因此營運成本高達85億美元。我認為openAI目前的問題在於大型語言模型訓練需要企業投入高成本，但商業化的未來卻不明確，因此該如何多方面的融入各個企業是openAi需要克服的問題。

**Comment by hunteryang0108** [comment]
我覺得這篇文章有點把科技發展以及企業經營混為一談了，我毫不懷疑地認為AI就是開創下一個時代的技術，OpenAI作為這個行業雖是作為這個行業的先驅，但也僅僅只佔了AI市場的一塊，其他仍有許多經營得不錯的公司在投入AI產業，所以並不能只以OpenAI的情況來評論整個AI的發展。
